config/printer-twotrees-sapphire-pro-2020.cfg

```diff
@@ -18,7 +18,8 @@ step_pin: PE3
 dir_pin: !PE2
 enable_pin: !PE4
 microsteps: 16
-rotation_distance: 32
+rotation_distance: 40
+#rotation_distance: 32
```

Comment:
Why keep the old rotation distance as a comment?  Does this mean there are multiple flavors of the Sapphire Pro?  If so, would a comment explaining when to use what not make more sense than just a second value in a comment?